src/components/MovieDetails.jsx

```diff
@@ -26,19 +30,13 @@ const MovieDetails = ({poster_path, title, vote_average, genres, release_date, r
 
                 <p className='movie-details__overview'>{overview}</p>
             </div>
-            <button className="reset-header-btn" onClick={ ()=> resetHeader("") }> &#x1F50E;&#xFE0E;</button>
+            <Link to={`/${location.search}`} className="reset-header-btn"> &#x1F50E;&#xFE0E;</Link>
         </div>
     )
 }
 
-MovieDetails.propTypes = {
-    poster_path: PropTypes.string,
-    title: PropTypes.string,
-    vote_average: PropTypes.number,
-    genres: PropTypes.arrayOf(PropTypes.string),
-    release_date: PropTypes.string,
-    runtime: PropTypes.number,
-    overview: PropTypes.string,
-}
-
 export default MovieDetails;
+
+export async function loader({ params}) {
```

Comment:
add type for params

Reply:
Module 10 has task to move all code to typescript